website/docs/docs/dbt-versions/core-upgrade/05-upgrading-to-v1.10.md

```diff
@@ -49,6 +49,15 @@ You can read more about each of these behavior changes in the following links:
 
 - (Introduced, disabled by default) [`validate_macro_args`](/reference/global-configs/behavior-changes#macro-argument-validation). If the flag is set to `True`, dbt will raise a warning if the argument `type` names you've added in your macro YAMLs don't match the argument names in your macro or if the argument types aren't valid according to the [supported types](/reference/resource-properties/arguments#supported-types).
 
+### Deprecation warnings as errors
```

Comment:
Actually, I think we can remove this as a net-new section in the 1.10 upgrade guide since I've already included it here -> https://github.com/dbt-labs/docs.getdbt.com/pull/7239